models/sudoswap/ethereum/sudoswap_ethereum_schema.yml

```diff
@@ -222,4 +222,55 @@ models:
       - *block_number
       - *tx_from
       - *tx_to
-      - *unique_trade_id
\ No newline at end of file
+      - *unique_trade_id
+
+  - name: sudoswap_ethereum_pools
+    meta:
+      blockchain: ethereum
+      project: sudoswap
+      contributors: [niftytable]
+    config:
+      tags: ['ethereum','sudoswap','pools']
+    description: >
+        sudoswap pools on Ethereum
+    columns:
+      - name: pool_address
+        description: "Address of the pool"
+      - name: nftcontractaddress
```

Comment:
minor nit: can we ensure all column names are using '_' to separate words? this would reflect here and the query in the model. there are a few instances here, but i'll just leave the comment here

Reply:
will make those changes now